cpp/src/barretenberg/benchmark/honk_bench/benchmark_utilities.hpp

```diff
@@ -0,0 +1,224 @@
+#include "barretenberg/crypto/ecdsa/ecdsa.hpp"
```

Comment:
It's not pressing now, but this file could be split into a cpp and hpp file, so that all instantiations of benchmarks don't reinclude all these hpp files

Reply:
Leaving this as a TODO. I'm sure I'll be mucking around in here again soon